exporter/loadbalancingexporter/trace_exporter_test.go

```diff
@@ -581,6 +581,16 @@ func simpleConfig() *Config {
 	}
 }
 
+func attrBasedRoutingConfig() *Config {
```

Comment:
If this is only used by the log exporter, shouldn't it be there instead of in the trace exporter ?